README.md

```diff
@@ -28,22 +53,53 @@ Backpointers
 This proposal is based on the LDML spec, List Patterns:
 
 
-### Prior Art
+## Proposal
 
 
-### Usage
+### Spec
 
-```javascript
-let o = new Intl.ListFormat("en", {
-    style: "regular" // default style
-});
-console.log(o.format(['foo', 'bar', 'baz']); // "foo, bar, and baz"
-```
+You can view the [spec text](spec/listformat.html).
+
+
+### API
+
+#### `Intl.ListFormat([locales[, options]])`
+
+The `Intl.ListFormat` object is a constructor for objects that enable language-sensitive list formatting.
+
+#### `locales`
+
+Optional. A string with a BCP 47 language tag, or an array of such strings. For the general form and interpretation of the `locales` argument, see the [`Intl` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
+
+#### `options`
+
+Optional. An object with some or all of the following properties:
+
+##### `localeMatcher`
+
+The locale matching algorithm to use. Possible values are `"lookup"` and `"best fit"`; the default is `"best fit"`. For information about this option, see the  [`Intl` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation).
+
+##### `type`
+
+
+The format of output message. Possible values are  `"conjunction"` that stands for "and"-based lists (default, e.g., `A, B, and C`), or `"disjunction"` that stands for "or"-based lists (e.g., `A, B, or C`). `"unit"` stands for lists of values with units (e.g., `5 pounds, 12 ounces`).
```

Comment:
Could you make this a bulleted list, here and in in MDN, for more legibility (here and whenever enumerating options)?